network-api/networkapi/buyersguide/templates/buyersguide/bg_base.html

```diff
@@ -0,0 +1,112 @@
+  <meta property="og:image:width" content="1200">
+  <meta property="og:image:height" content="628">
+  <meta property="og:url" content="{{request.scheme}}://{{request.get_host}}{{request.path}}">
+{% endblock %}
+
+
+{% block stylesheets %}
+  <link rel="stylesheet" href="{% static "_css/buyers-guide.compiled.css" %}">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Nunito+Sans:400,300,700,300i,400i">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Zilla+Slab:300,400,600,700,300i,400i">
+{% endblock %}
+
+
+{% block icons %}
+  <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="{% static "_images/favicons/apple-touch-icon-180x180@2x.png" %}">
+  <link rel="icon" type="image/png" sizes="196x196" href="{% static "_images/favicons/favicon-196x196@2x.png" %}">
+  <link rel="shortcut icon" href="{% static "_images/favicons/favicon.ico" %}">
+{% endblock %}
+
+
+{% block additional_head_elements %}
+  {% if web_monetization_pointer %}
+    <meta name="monetization" content="{{ web_monetization_pointer }}">
+  {% endif %}
+  <!-- TODO: determine if we still need this -->
+  <script>
+    var modernBrowser = (
+      'fetch' in window &&
+      'assign' in Object
+    );
+
+    if ( !modernBrowser ) {
+      var scriptElement = document.createElement('script');
+      scriptElement.async = false;
+      scriptElement.src = '{% static "_js/polyfills.compiled.js" %}';
+      document.head.appendChild(scriptElement);
+    }
+  </script>
+{% endblock %}
+
+
+{% block bodyclass %}pni{% endblock %}
+
+
+{% block script_bundle %}
+  <script src="{% static "_js/bg-main.compiled.js" %}" async defer></script>
+{% endblock %}
+
+
+{% block body_wrapped %}
+
+  <div class="primary-nav-container-wrapper">
+  {% include "buyersguide//primary_nav.html" %}
+  </div>
+
+  {% block hero %}{% endblock %}
+
+  {% block catgory_nav %}
+  {% include "../partials/multipage_nav_mobile.html" with pni_category_nav=True wrapper_classes="mt-0" %}
+  <nav id="multipage-nav" class="pni-category-nav text-center d-none d-md-block" title="{% trans "site navigation" context "Tooltip on menu items" %}">
+    <div class="container">
+      <div class="row">
+        <div class="col">
+          {% if pagetype == "product" or pagetype == "about" %}
+            <a class="multipage-link active" href="{{ page.url }}">{% trans "All" %}</a>
+          {% else %}
+            <a class="multipage-link {% if not category %} active{% endif %}" href="{{ page.url }}">{% trans "All" %}</a>
+          {% endif %}
+          {% for cat in categories %}
+            {% if cat.published_product_count > 0 %}
+              {% routablepageurl page 'category-view' cat.slug as cat_url %}
+              <a class="multipage-link {% check_active_category current_category cat %}{% if cat.featured is True %} featured{% endif %}{% if category == cat.slug %} active{% endif %}" href="{{ cat_url }}" data-name="{{ cat.name }}">{{ cat.name }}</a>
```

Comment:
Instead of using an `include` here I opted to _not_ copy that file and use a for loop here. The `cat.published_product_count` came with the original template.